lib/src/geo/latlng_bounds.dart

```diff
@@ -1,86 +1,125 @@
-import 'dart:math' as math;
+import 'dart:math';
 
 import 'package:latlong2/latlong.dart';
 import 'package:vector_math/vector_math_64.dart';
 
 /// Data structure representing rectangular bounding box constrained by its
 /// northwest and southeast corners
 class LatLngBounds {
-  late LatLng _sw;
-  late LatLng _ne;
+  /// The latitude north edge of the bounds
+  double north;
+
+  /// The latitude south edge of the bounds
+  double south;
+
+  /// The longitude east edge of the bounds
+  double east;
+
+  /// The longitude west edge of the bounds
+  double west;
+
+  /// Create a [LatLngBounds] instance from raw edge values.
+  ///
+  /// Potentially throws assertion errors if the coordinates exceed their max
+  /// or min values or if coordinates are meant to be smaller / bigger
+  /// but aren't.
+  LatLngBounds({
```

Comment:
Instead of making this the new default constructor, can we call this `fromRawEdges`, and make the `fromCorners` un-named again? Just trying to minimize this breaking change.

Reply:
Can do, but instead of `fromRawEdges` I'd use the `unsafe` to be more in line with `Bounds.unsafe`. Sounds alright?